py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
Apologies. I'm doing things wrong again. Sorry. I can python3-test modules using...
```
lasers~/src/py3status/py3status/modules git:(master) python3 nvidia_temp.py
{'cached_until': 1485521800.0, 'color': '#00FF00', 'full_text': 'GPU: 62°C'}
{'cached_until': 1485521801.0, 'color': '#00FF00', 'full_text': 'GPU: 62°C'}
```

For some unexplainable reason, I tried `py3status-python2 nvidia_temp.py` when I really should try `python2 nvidia_temp.py`. It seems like I will have to use `py3status2` in the i3 config instead to actually test py3status w/ python2 instead of just running things from terminal.